app/helpers/name_authority_helper.rb

```diff
@@ -0,0 +1,29 @@
+require 'redis'
```

Comment:
This is super picky but the combination of space and tab indentation makes this hard to read. I think the standard we've used throughout is two spaces to indent.
<img width="511" alt="Screen Shot 2021-04-22 at 1 58 22 PM" src="https://user-images.githubusercontent.com/458247/115764067-1becc380-a373-11eb-8fd4-9d02882a3d23.png">